packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -423,18 +419,24 @@ class Focus extends StatefulWidget {
   /// widget, or there is no [Focus] widget in scope, then this method will
   /// return null.
   ///
-  /// The `context` and `scopeOk` arguments must not be null.
-  ///
-  /// Calling this function creates a dependency that will rebuild the given
-  /// context when the focus changes.
+  /// {@template flutter.widgets.focus_scope.Focus.maybeOf}
+  /// If `createDependency` is true (which is the default), calling this
+  /// function creates a dependency that will rebuild the given context when the
+  /// focus changes.
```

Comment:
This doc leaves me wondering what constitutes a "focus change": Does it only rebuild when maybeOf will return a different FocusNode instance? Or does it (also) rebuild when some property (what?) on the FocusNode changes?

Reply:
This is basically just any time the primaryFocus changes. Updated the docs to be more clear.